lib/venue.rb

```diff
@@ -1,2 +1,30 @@
 class Venue
+  attr_reader :name, :capacity, :patrons
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+  def add_patron(name)
+    @patrons << name
+  end
+  def yell_at_patrons
+    @patrons.map! { |name| name.upcase }
+  end
+
+  def over_capacity?
+    if @patrons.size <= @capacity
+      return false
```

Comment:
Nice use of the `if/else` here. We could simplify this by just comparing `@patron.size > @capacity` to get our `true or false`. That comparison will return true or false. And we would want our `@patrons` array to be greater than our `@capacity`.